daemon/cmd/ipam_test.go

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+//go:build !privileged_tests
```

Comment:
Looks like this needs an old `// +build` comment too given we still build `v1.10` using Go 1.17, see https://github.com/cilium/cilium/runs/8224150412?check_suite_focus=true#step:8:90

Reply:
Thanks, fixed.